Manifests/ManifestsApple/com.apple.applicationaccess-tvOS.plist

```diff
@@ -17,7 +17,7 @@
 	<key>pfm_interaction</key>
 	<string>combined</string>
 	<key>pfm_last_modified</key>
-	<date>2022-02-09T09:11:11Z</date>
+	<date>2022-09-05T01:03:41Z</date>
```

Comment:
bump pfm_version as well?

Reply:
Keys weren't added or removed in this manifest.